specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json

```diff
@@ -4332,7 +4332,7 @@
       "type": "object",
       "allOf": [
         {
-          "$ref": "#/definitions/TrackedResource"
+          "$ref": "#/definitions/ProxyResource"
```

Comment:
Can I know why have this change from business/service perspective? as it is a breaking change for a stable api version

Reply:
It should have always been a proxy resource - it is associated with an endpoint which is a tracked resource. We don't need location/tags for this resource. As I said, this API version and the ones following it are not released as yet.